pkg/server/webhook.go

```diff
@@ -205,6 +195,14 @@ func checkIfNeedInjectByConfig(injectionConfig *config.InjectionConfig, labels m
 		}
 	}
 
+	if injectionConfig.InjectAll {
+		requestedInjection, ok := labels[requestAnnotationKey]
+		if ok {
+			return true, requestedInjection
+		} else {
+			return true, DefaultSideCarKey
```

Comment:
else not needed here.
Removing it will make it more obvious that this function returns here in all cases of `ok`.